CONTRIBUTING.md

```diff
@@ -78,6 +79,7 @@ Go to the root of the repository and run the setup:
 
 ```bash
 cd strapi
+# NOTE: If you don't already have node_modules, you will need to run `yarn install` first.
```

Comment:
What do you think about removing this comment and always suggest to run `yarn install` before? It almost costs nothing and I think would remove some ambiguity.

Reply:
```suggestion
yarn install
```